libs/ddhub-client-gateway-message-broker/src/lib/services/ddhub-base.service.ts

```diff
@@ -101,6 +102,23 @@ export abstract class DdhubBaseService {
       );
     }
 
+    if (
```

Comment:
it will kill re-authentication as it will not ever trigger `      await this.ddhubLoginService.login(false);
`

Reply:
why MB cannot throw Forbidden instead of unauthorized? flow would be much simpler